dracut/30ignition/clevis-decrypt-sss-wrapper

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+LD_LIBRARY_PATH=/sysusr/usr/lib64 exec /sysusr/usr/bin/clevis-decrypt-sss "$@"
```

Comment:
I suggested this approach but at that time already wanted the final form to be symlinks to a single script where `basename "$0"` is used. Would be nice to do this now before adding so many helpers.

Reply:
Yes, good point.

To reduce duplication and make this wrapping functionality as easy to use as possible, I thought it would be nice to have a spot somewhere in the Ignition module's `module-setup.sh` that one can add a path to to and then an appropriate entry in /usr/bin/$executable will appear. That is, it would be nice to auto-generate the entries in `/usr/bin`.

In the changes I just pushed, I decided to autogenerate wrappers rather than symlinks to generic wrappers with `basename "$0"`. If we auto-generate the symlink or wrapper anyways, then we might as well just generate a wrapper rather than a generic wrappers and the symlink. (Note we'd need more than one generic wrapper: we'd need one wrapper that execs `/sysusr/usr/bin/$(basename "$0")` and another one that execs `/sysusr/usr/sbin/$(basename "$0")` etc.) If that reasoning doesn't make sense, let me know and I'd gladly switch to symlinks :) 

Since this change removed most of the files added in earlier commits, I force-pushed to make the PR easier to read.